cfgov/jinja2/v1/the-bureau/bureau-structure/_macro-content-slider.html

```diff
@@ -0,0 +1,35 @@
+{# ==========================================================================
+
+   content-slider.render()
+
+   ==========================================================================
+
+   Description:
+
+   Renders a content slider for the given categories.
+
+   categories: An Array of categories.
+
+   ========================================================================== #}
+{% macro render(categories) -%}
+<div id="content-slider-container">
```

Comment:
Is `content-slider-container` id still needed? There's an old issue around the nested ids here https://github.com/cfpb/cfgov-refresh/issues/687